src/components/FileRepo/FileRepo.js

```diff
@@ -6,13 +6,16 @@ import { isObject } from 'lodash';
 import { Trans } from 'react-i18next';
 import FilterIcon from 'react-icons/lib/fa/filter';
 
+import Tooltip from 'uikit/Tooltip';
+import CavaticCopyButton from 'components/cavatica/CavaticaCopyButton';
```

Comment:
CavaticaCopyButton?

Reply:
hawk eye!

Reply:
@nyanofthemoon it's just a misspelling :P

Reply:
Ohhhhhh totally missed that one. 🥇 Fixing.